R/tax_range_time.R

```diff
@@ -0,0 +1,178 @@
+#' Calculate the temporal range of fossil taxa
+#'
+#' A function to calculate the temporal range of fossil taxa.
+#'
+#' @param occdf \code{dataframe}. A dataframe of fossil occurrences. The
+#' dataframe should contain at least three columns: names of taxa (`character`)
+#' and the maximum (`numeric`) and minimum (`numeric`) age which constrain the
+#' age range (in millions of years) of the fossil occurrence. If the age range
```

Comment:
```suggestion
#' age range of the fossil occurrence. If the age range
```

Reply:
I think we should retain this. If the data is not provided in millions of years the plotting functionality will behave oddly!